pmm/pmm2-submodules.groovy

```diff
@@ -21,7 +21,7 @@ void addComment(String COMMENT) {
             curl -v -X POST \
                 -H "Authorization: token ${GITHUB_API_TOKEN}" \
                 -d "{\\"body\\":\\"${COMMENT}\\"}" \
-                "https://api.github.com/repos/\$(echo $CHANGE_URL | cut -d '/' -f 4-5)/issues/${CHANGE_ID}/comments"
+                "https://api.github.com/repos/\$(echo ${CH_URL} | cut -d '/' -f 4-5)/issues/${CH_ID}/comments"
```

Comment:
why do we need to change variable name?

Reply:
@EvgeniyPatlan We have converted pmm2-submodules/pmm3-submodules from multibranch pipeline to pipeline, hence parameters i.e. CH_URL/CH_ID have to be passed from multibranch pipeline job "pmm-submodules" to the individual pipelines. There were some issues with using the same variable name i.e. "CHANGE_ID" since it was set to a different value, hence changed the variable names.

Reply:
Reverted variable names back to the original name and things work fine now. Thank you for raising this point @EvgeniyPatlan